public/app/features/panel/query_ctrl.ts

```diff
@@ -10,8 +10,10 @@ export class QueryCtrl {
   error: string;
   isLastQuery: boolean;
 
+  /** @ngInject */
   constructor(public $scope: any, public $injector: auto.IInjectorService) {
-    this.panel = this.panelCtrl.panel;
+    this.panel = this.panelCtrl?.panel ?? this.$scope.ctrl.panelCtrl.panel;
+    this.target = this.panelCtrl?.target ?? this.$scope.ctrl.panelCtrl.target;
```

Comment:
@torkelo -- things appear to almost work when the values are set from: `this.$scope`, but the properties that used to bind automatically do not appear to be binding